multiapps-mta/src/main/java/org/cloudfoundry/multiapps/mta/resolvers/v2/ResourceLiveParameterResolver.java

```diff
@@ -0,0 +1,149 @@
+package org.cloudfoundry.multiapps.mta.resolvers.v2;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.cloudfoundry.multiapps.common.util.MiscUtil;
+import org.cloudfoundry.multiapps.mta.model.DeploymentDescriptor;
+import org.cloudfoundry.multiapps.mta.model.Module;
+import org.cloudfoundry.multiapps.mta.model.ProvidedDependency;
+import org.cloudfoundry.multiapps.mta.model.Resource;
+import org.cloudfoundry.multiapps.mta.resolvers.Reference;
+import org.cloudfoundry.multiapps.mta.resolvers.ReferencePattern;
+import org.cloudfoundry.multiapps.mta.resolvers.Resolver;
+
+public class ResourceLiveParameterResolver implements Resolver<Resource> {
+
+    private static final String RESOURCE_PARAMETER_KEY_CONFIG = "config";
+    private final DeploymentDescriptor deploymentDescriptor;
+    private final Map<String, String> idleToLiveParameterPairs;
+    private Resource resource;
+
+    public ResourceLiveParameterResolver(DeploymentDescriptor deploymentDescriptor, Resource resource,
+                                         Map<String, String> idleToLiveParameterPairs) {
+        this.deploymentDescriptor = deploymentDescriptor;
```

Comment:
I also do not see if the resource properties are getting resolved?
A resource can have properties that are getting injected in the required module